### PR TITLE
feat(rfp): alternative javascript console for polkadot js API

### PR DESCRIPTION
**Pull request by muddlebee**
# Request for Proposals

## Abstract

Alternative polkadot js API playground

## Checklist

- [x] I have checked the [RFPs](https://github.com/w3f/Grants-Program/tree/master/docs/rfps) to make sure this is not a duplicate.
- [x] I have read and followed the [RFP Suggestion instructions](https://github.com/w3f/Grants-Program#mailbox_with_mail-suggest-a-project).

### CODE REVIEW

**Comment by muddlebee** [comment]
wrt #1235 
@0xCaso

**Review by 0xCaso — COMMENTED** [review]
Thanks for the re-submission @muddlebee.

As I wrote in the original issue, could you please re-format the document following the [RFP template](https://github.com/w3f/Grants-Program/blob/master/docs/RFPs/suggestion-template.md)?

**Comment by muddlebee** [comment]
@0xCaso apologies. missed it. 
I have fixed it now according to the format. Any suggestions/improvements to this RFP please let me know 🥂

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks a lot for creating the RFP. I have one question/comment: Would you be interested in using this new javascript console personally or as part of a project, and if so, why? Maybe this will help to show the usefulness of the RFPs also to others.

**Review by Noc2 — APPROVED** [review]
Thanks for the quick reply. I'm generally happy to merge the RFP. But maybe you can also integrate these use cases into the RFP itself.

**Review by semuelle — CHANGES_REQUESTED** [review]

**Review by semuelle — COMMENTED** [review]
Getting (significant) changes merged into `apps` has proven difficult in the past. I think this RFP should either be standalone or contributed to some other established project.

**Comment by muddlebee** [comment]
@semuelle my suggestion for this RFP was meant to be a standalone project as I have mentioned in the title of the project.
> Alternative javascript console for Polkadot JS API

Please let me know if I need to clarify more about the details in the RFP?

**Review by Noc2 — DISMISSED** [review]

**Comment by muddlebee** [comment]
@Noc2 I have mentioned the use cases towards the end of the RFP doc at the Deliverables section. Please let me know if I need to clarify more?

**Review by semuelle — APPROVED** [review]
Great, thanks.

File: docs/RFPs/Open/alternative-polkadot-js-api-console.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+**Current limitations of the above console**
+Cannot save code properly, not much keyboard shortcuts, cannot customize configurations.
+
+**Alternative polkadot js API playground**
+1. [https://github.com/subdirectory/subshell](https://github.com/subdirectory/subshell)
+
+
+### Proposed-RFP
+
+A new polkadot js API playground with vs-code like configurations like save the code, workspace, keyboard shortcuts etc etc..
```
Comment (keeganquigley) [inline docs/RFPs/Open/alternative-polkadot-js-api-console.md:27]:
```suggestion
A new Polkadot-JS API playground with VS Code-like configurations like save the code, workspace, keyboard shortcuts, etc.
```

File: docs/RFPs/Open/alternative-polkadot-js-api-console.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+[https://polkadot.js.org/apps/#/js](https://polkadot.js.org/apps/#/js)
+
+some examples -> https://playground.substrate.dev/
+here we have to manually build and run our js bundles
+![image](https://user-images.githubusercontent.com/8139783/198254152-abdd0f2e-62d4-4f0f-aad1-bcfdd6d48a74.png)
+
+**Why alternative javascript console for for polkadot js API**
```
Comment (keeganquigley) [inline docs/RFPs/Open/alternative-polkadot-js-api-console.md:34]:
```suggestion
**Why alternative javascript console for Polkadot-JS API**?
```

File: docs/RFPs/Open/alternative-polkadot-js-api-console.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  - study how the current javascript console is developed at https://polkadot.js.org/apps/#/js
+  - understand the libaries currently integrated from [polkadot JS API docs](https://polkadot.js.org/docs/)
+
+- Development:
+  - design a new UI/UX with better experience than current javascript console with features like
+    - save code preferably with secure session management
+    - keyboard shortcuts
+    - [example](#Proposed-RFP)
+    
+    more features which makes the polkadot js experience productive and smoother are welcome
```
Comment (keeganquigley) [inline docs/RFPs/Open/alternative-polkadot-js-api-console.md:54]:
```suggestion
    Any additional features which make the Polkadot-JS experience more productive and smoother are welcome.
```

**Review by keeganquigley — CHANGES_REQUESTED** [review]
Thanks for the submission @muddlebee, I made a few minor edit suggestions to clean up the language a bit.

**Review by randombishop — DISMISSED** [review]
LG

**Comment by muddlebee** [comment]
@keeganquigley thank you for the suggestions. I have committed the changes as per the request 🎉

**Review by semuelle — APPROVED** [review]

**Review by Noc2 — APPROVED** [review]

**Review by keeganquigley — APPROVED** [review]

**Pull request merged**